pallets/uniques/src/benchmarking.rs

```diff
@@ -0,0 +1,181 @@
+// This file is part of Substrate.
```

Comment:
What file did we copy? Did not all the code change?

Reply:
It should be removed.

Reply:
https://github.com/NodleCode/chain/pull/753/commits/798601553e8565df5de74c276725c8e417ca9605 implements it